vagrant-spk

```diff
@@ -395,6 +395,86 @@ mkdir -p /var/run
 /usr/sbin/nginx -g "daemon off;"
 """
 
+DIY_SETUP_SCRIPT = r"""#!/bin/bash
+# This script is run in the VM once when you first run `vagrant-spk up`.  It is
```

Comment:
I propose we add a one-line summary to the top of this, e.g.:

`Purpose: Install dependencies needed to run the app but that don't change when the source code changes`

But is that an accurate summary? In a way, the only difference between the `setup` and `build` phases is that `setup` is only run once, which seems to me to be an optimization.

I wonder if there's a clearer way to express this, and/or if you agree with the `Purpose: ...` line I named above.